src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -31,9 +31,7 @@
 from libs import handler
 from metrics import logs
 
-HIGH_TEMPERATURE_PARAMETER = .5
-MEDIUM_TEMPERATURE_PARAMETER = .25
-LOW_TEMPERATURE_PARAMETER = .15
+TEMPERATURE_PARAMETER = .15
```

Comment:
Could you leave a comment explaining why we went with this temperature?

Reply:
Sure.